lib/components/artist/shows/metadata.js

```diff
@@ -55,8 +54,8 @@ export default Relay.createContainer(Metadata, {
       fragment on PartnerShow {
         kind
         name
-        start_at
-        end_at
+        exhibition_period
+        status
```

Comment:
I want to add:
```
counts: {
  artworks(artist_id: ARTIST_ID)
}
```

but I'm not sure what the right form is since that would require the artist ID to reach this fragment somehow.